pages/common/git-grep.md

```diff
@@ -0,0 +1,19 @@
+# git-grep
+
+> Find strings inside files as they existed anywhere in repo history. Accepts a lot of the same flags as regular grep. Learn more here: https://git-scm.com/docs/git-grep.
```

Comment:
Perhaps this will read better?

```suggestion
> Find strings inside files anywhere in a repository's history.
> Accepts a lot of the same flags as regular `grep`.
> Documentation: https://git-scm.com/docs/git-grep.
```

Reply:
@sbrl, does this need the angle brackets around the URL?

Reply:
Yes. I forgot :P